switchmap/poller/snmp/snmp_manager.py

```
@@ -170,48 +190,36 @@ def hostname(self):
         # Try to reach device
         try:
-            # If we can poll the SNMP sysObjectID,
-            # then the device is contactable
-            result = self.sysobjectid(check_reachability=True)
+            # Test if we can poll the SNMP sysObjectID
+            # if true, then the device is contactable
+            result = await self.sysobjectid(check_reachability=True)
             if bool(result) is True:
                 contactable = True
 
         except Exception:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)